.changeset/moody-stingrays-add.md

```diff
@@ -0,0 +1,6 @@
+---
+"@medusajs/medusa": patch
+"integration-tests-api": patch
```

Comment:
todo: Remove patch bump of integration tests

Reply:
Done.